### ISSUE TITLE
Incompatible with 0.5.1c version of create

### ISSUE THREAD

**Issue by Diemoerm** [Layers-of-Railways/Railway#203]
When used together with version 0.5.1c, building a train can cause a crash

**Comment by zkeepah** [comment]
I believe I have the same issue. On the same version of create I find that the game crashes if you change axle style on the train bogies or try to place a monorail bogie. 
It also crashes when attempting to load a world where a monorail bogie is already placed from before I've updated create. 

I've got this snippet from the game crash here: 

The game crashed whilst rendering block entity
Error: java.lang.NoSuchMethodError: 'com.jozufozu.flywheel.util.transform.Transform com.railwayteam.railways.content.custom_bogeys.monobogey.MonoBogeyRenderer$SmallMonoBogeyRenderer.getTransformFromPartial(com.jozufozu.flywheel.core.PartialModel, com.mojang.blaze3d.vertex.PoseStack, boolean)'

And I've also attached a copy of the full crash report.
[crash.txt](https://github.com/Layers-of-Railways/Railway/files/12206773/crash.txt)
